frontend/src/components/table/Table.tsx

```diff
@@ -0,0 +1,10 @@
+import { Table as ATable } from "antd";
+import styled from "styled-components";
+
+import { Colors } from "../../styles";
+
+const Table = styled(ATable)`
+    
```

Comment:
Tutaj chyba będzie kilka rzeczy do ostylowania, wrzuć jakieś dane do tabelki i to sprawdź